### ISSUE TITLE
Not able to import Warranty Details of Dell Assets through Manufacturer Import

### ISSUE THREAD

**Issue by chintan-pansuriya** [InfotelGLPI/manufacturersimports#51]
Dear The Plugins Team,

The Plugin team.

------------
* Version of the plugins : Manufacturer Import Version 2.2

* Version of your GLPI : 9.2

* Steps to reproduce (which actions have you made) :
I have already taken dell api key and dell team was given me api key as well as url address of sandbox for texting purpose.
Same i have configured the key on application menu. Now, whenever i have clicked on manufacturer information than next webpage opening as xml and showing details but not found any option for import details.

Kindly some one guide to me.

**Comment by amma35** [comment]
Hi,
Import for each machine in massive action to add administrative information.

**Comment by psychomoise** [comment]
hi

I got the same issue as I just update GLPI and I can use this pretty good plugin.
The issue is a limit into the plugin where the field that is storing the API key is limited to 32 chars, but dell API key are longer than this.

you just need to update the field to get it a bit larger, here is the SQL request for MySql : 
ALTER TABLE `glpi_plugin_manufacturersimports_configs` CHANGE `supplier_key` `supplier_key` VARCHAR(64) CHARACTER SET utf8 COLLATE utf8_unicode_ci NULL DEFAULT NULL;

after thatI , I was able to update the key in the plugin through the web interface and the key was correct.
I did a test on their sandbox API and it worked like a charm.

I am now waiting for Dell to promote the key into production so I can use it completly.
